autoswitch_virtualenv.plugin.zsh

```diff
@@ -81,7 +81,8 @@ function _check_venv_path()
         printf "${check_dir}/.venv"
         return
     else
-        if [ "$check_dir" = "/" ]; then
+        # Abort search at file system root or HOME directory.
+        if [[ "$check_dir" = "/" || "$check_dir" = "$HOME" ]]; then
```

Comment:
is this mainly for performance purposes or is there something else I'm not understanding?

Reply:
Performance really ; there shouldn't be a need to check outside of a users home if it has started in there.

Reply:
Could you update the comment to mention "This is purely for performance purposes"?

Reply:
Done!